src/display.rs

```diff
@@ -11,16 +11,43 @@
 
 use libc;
 use base::CGError;
+use geometry::{CGRect, CGPoint, CGSize};
 
 pub type CGDirectDisplayID = libc::uint32_t;
 
 #[link(name = "ApplicationServices", kind = "framework")]
 extern {
     pub fn CGMainDisplayID() -> CGDirectDisplayID;
+    pub fn CGDisplayIsActive(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsAlwaysInMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsAsleep(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsBuiltin(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsInHWMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsInMirrorSet(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsMain(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsOnline(display: CGDirectDisplayID) -> bool;
+    pub fn CGDisplayIsStereo(display: CGDirectDisplayID) -> bool;
```

Comment:
Indeed it seems like it's `u32` on x64 and `i32` on x86. I considered adding
```rust
#[cfg(target_pointer_width = "32")]
pub type boolean_t = i32;
#[cfg(target_pointer_width = "64")]
pub type boolean_t = u32;
```
but it feels like this would be better suited elsewhere. Any ideas where this could go instead?